images/metrics_fileinfo/Dockerfile.erb

```
@@ -0,0 +1,12 @@
+FROM sider/devon_rex_base:2.28.0
+
+# install file command (file type guesser)
+USER root
+RUN apt-get update -y && \
+    apt-get install -qq -y --no-install-recommends file="1:5.35-4+deb10u1" && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+USER $RUNNER_USER
```

I think `file` command is so stable that we can always use the latest version. Can we make the version relaxed as follows?

```suggestion
USER root
RUN apt-get update && \
    apt-get install -y --no-install-recommends file && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
USER $RUNNER_USER
```

This is the exact copy from our `shellcheck` [Dockerfile](https://github.com/sider/runners/blob/master/images/shellcheck/Dockerfile.erb).